pkg/options/types_test.go

```diff
@@ -202,13 +202,48 @@ func TestLabelsAllowListSet(t *testing.T) {
 					"bar",
 					"*"}}),
 		},
+		{
+			Desc:  "with quoted tokens",
```

Comment:
Can you add one negative test with a non-matching quotation?

something like `"node=["kubernetes.io/arch]"`

Reply:
Good point. Since we do not validate the expression syntactically, such a test case would actually pass. It seems that the quotes are simply skipped during processing. 

Is this something we want to tackle as well?

Reply:
I don't have a strong opinion here, maybe we can also use an upstream parser for this? https://github.com/kubernetes/apimachinery/tree/master/pkg/labels

Reply:
I looked into the labels parser and it also does not accept quotation marks. On the other hand, it will explicitly fail with:
```
Invalid value: "\"nodes": name part must consist of alphanumeric characters
```
We could also do the same as an alternative to disregarding quotes.

Reply:
My preference would be to match the behavior of what's in Kubernetes